src/sentry/api/endpoints/organization_dashboard_details.py

```diff
@@ -196,6 +202,18 @@ def post(self, request: Request, organization, dashboard) -> Response:
         dashboard.last_visited = timezone.now()
         dashboard.save(update_fields=["visits", "last_visited"])
```

Comment:
This stuff will be phased out when we switch to this user-centric visit measure, so it's inconsequential that it's set before I search for the org member.